### PR TITLE
Add media_player platform to Android TV Remote

### PR DESCRIPTION
**Pull request by Drafteed**
## Proposed change

Docs related to core PR: https://github.com/home-assistant/core/pull/91677

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/91677
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by bdraco — APPROVED** [review]

File: source/_integrations/androidtv_remote.markdown [hunk 1]
```diff
@@ -44,6 +43,132 @@ Examples of URLs to pass as activity for some popular apps:
...
+The remote allows you to send key commands to your Android TV device with the `remote.send_command` service.
+The entity has the `current_activity` attribute that shows the current foreground app on the Android TV.
+
+{% details "List of the most common commands" %}
+
+Navigation:
+- DPAD_UP
+- DPAD_DOWN
+- DPAD_LEFT
+- DPAD_RIGHT
```
Comment (frenck) [inline source/_integrations/androidtv_remote.markdown:95]: This is unrelated? Why was this changed in this PR?

File: source/_integrations/androidtv_remote.markdown [hunk 2]
```diff
@@ -44,6 +43,132 @@ Examples of URLs to pass as activity for some popular apps:
...
+- CAPTIONS
+- DVR
+- MEDIA_AUDIO_TRACK
+- SETTINGS
+- SEARCH
+- ASSIST
+
+{% enddetails %}
+
+For a full list see [here](https://github.com/tronikos/androidtvremote2/blob/main/src/androidtvremote2/remotemessage.proto#L90).
```
Comment (frenck) [inline source/_integrations/androidtv_remote.markdown:168]: We should not link to source code, please remove this.

File: source/_integrations/androidtv_remote.markdown [hunk 3]
```diff
@@ -44,6 +43,132 @@ Examples of URLs to pass as activity for some popular apps:
...
 | Disney+ | https://www.disneyplus.com
 
+Examples:
+
+```yaml
+# Launch the Netflix app
+service: media_player.play_media
+data:
+  media_content_type: url
+  media_content_id: https://www.netflix.com/title
```
Comment (frenck) [inline source/_integrations/androidtv_remote.markdown:53]: These should have been quoted, see our YAML styling guides on our developer documentation for more information.

Please adjust this in a new PR (same for all other examples).

File: source/_integrations/androidtv_remote.markdown (hunk 1 above)
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown:95]: I was changing the structure of the documentation to make the docs common to both the media player and remote platforms, and I forgot to mention that I moved the list of commands from an external link into the documentation. Sorry for that. In any case, this should have been changed considering your next comment.

File: source/_integrations/androidtv_remote.markdown (hunk 2 above)
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown:168]: This was before my changes so I left it. 
In the new PR, I removed it: https://github.com/home-assistant/home-assistant.io/pull/27308

File: source/_integrations/androidtv_remote.markdown (hunk 3 above)
Comment (Drafteed) [reply source/_integrations/androidtv_remote.markdown:53]: Done in the new PR: https://github.com/home-assistant/home-assistant.io/pull/27308
